docs/source/1.0/features/feature-combination-matrix.md

```diff
@@ -1,18 +1,19 @@
+| EAGLE-3(One Model Engine)  | Yes               | Yes        | Yes                        | No                    | Untested        | No       | ---                       |                           |               |                  |                |                        |                       |                 |      |
+| EAGLE-3(Two Model Engine)  | No                | Yes        | Yes                        | No                    | Untested        | No       | No                        | ---                       |               |                  |                |                        |                       |                 |      |
+| Torch Sampler              | Yes               | Yes        | Yes                        | Yes                   | Yes             | Yes      | Yes                       | Yes                       | ---           |                  |                |                        |                       |                 |      |
+| TLLM C++ Sampler           | Yes               | Yes        | Yes                        | Yes                   | Yes             | No       | No                        | No                        | No            | ---              |                |                        |                       |                 |      |
+| KV Cache Reuse             | Yes               | Yes        | Yes                        | Untested              | Untested        | Untested | Yes                       | No                        | Yes           | Yes              | ---            |                        |                       |                 |      |
+| Slide Window Attention     | Yes               | Yes        | Yes                        | Untested              | Untested        | Untested | Untested                  | Untested                  | Yes           | Yes              | WIP            | ---                    |                       |                 |      |
+| Logits Post Processor      | No                | Yes        | Yes                        | No                    | Untested        | No       | No                        | No                        | Yes           | Yes              | Yes            | Yes                    | ---                   |                 |      |
+| Guided Decoding            | No                | Yes        | Yes                        | Untested              | Yes             | No       | No                        | No                        | Yes           | Yes              | Yes            | Yes                    | Yes                   | ---             |      |
+| LoRA                       | Yes               | Yes        |                            |                       |                 |          |                           |                           |               |                  |                |                        |                       |                 | ---  |
```

Comment:
Hi @shaharmor98 @Naveassaf , we are adding LoRA to the feature combination matrix, could you please help review this part? Can LoRA work with other features like Overlap Scheduler/CUDA Graph/Attention Data Parallelism/Disaggregated Serving/etc?